scripts/buildApiDocs/buildReference.ts

```diff
@@ -125,7 +125,19 @@ export async function buildReference(grep: RegExp | null = null) {
   rmSync(TEMP_DIR, { recursive: true });
 }
 
+function isObjectEmpty(object: Record<any, any>) {
+  // eslint-disable-next-line
+  for (const _ in object) {
+    return false;
+  }
+  return true;
+}
+
 function sortObjectByKeys<T>(obj: Record<string, T>, order: string[]): Record<string, T> {
+  if (isObjectEmpty(obj) || order.length === 0) {
```

Comment:
My fix was to update the order.json to:
```
  "cssVariables": [
    "__EVERYTHING_ELSE__"
  ],
```
with the thought that we will update this array later. This works too, but we probably wouldn't even use this function if the order array is empty (it means this data does not need to be sorted).